src/System.Numerics.Vectors/tests/Performance/Perf.Vector.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Xunit.Performance;
+
+namespace System.Numerics.Tests
+{
+    // Equals, DotProduct, ctor
+    public class Perf_Vector
```

Comment:
The most interesting things to test here are probably the methods under the "Intrinsics" files. Those are operations that are specially handled by the JIT. The most interesting ones are probably:

* Operator +, Operator -, Operator *, and the named versions, Add, Subtract, Multiply
* Dot()
* SquareRoot
* Length(Squared)
* Normalize
* Cross (for Vector3)
* Distance(Squared)

I'd also like to have them separated out by type (i.e. Vector2, Vector3, Vector4), because the code generated for them is probably different enough that we want to separate the results.